ui/src/tests/welcomePage.test.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect, vi } from 'vitest'
+import { mount } from '@vue/test-utils'
+import WelcomePage from '@/components/WelcomePage.vue'
+import { createTestingPinia } from '@pinia/testing'
+import { useUserStore } from '@/store/useUserStore'
+import useRouter from '@/composables/useRouter'
+
+const wrapper = mount(WelcomePage, {
+	global: {
+		plugins: [createTestingPinia()]
+	}
+} as any)
+
+test('Should redirect to situation list if not accept', async () => {
+	const router = useRouter()
+	const push = vi.spyOn(router, 'push')
+	const store = useUserStore()
+	store.userId = 'test'
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	//@ts-ignore
```

Comment:
I think these can be removed, and the ones below. At least no TS issues without them on my end